include/TPP/Passes.td

```diff
@@ -253,4 +253,12 @@ def PropagatePackUnPack : Pass<"propagate-pack-and-unpack", "func::FuncOp"> {
   let constructor = "mlir::tpp::createPropagatePackUnPackPass()"; 
 }
 
+def ConstantFold : Pass<"constant-fold", "ModuleOp"> {
```

Comment:
Too generic name, make sure it's clear it's pack specific

Reply:
Done.